mathesar/rpc/tables/metadata.py

```diff
@@ -77,26 +86,22 @@ def list_(*, database_id: int, **kwargs) -> list[TableMetaData]:
     """
     table_meta_data = get_tables_meta_data(database_id)
     return [
-        TableMetaData.from_model(model) for model in table_meta_data
+        TableMetaDataRecord.from_model(model) for model in table_meta_data
     ]
 
 
-@rpc_method(name="tables.metadata.patch")
+@rpc_method(name="tables.metadata.set")
 @http_basic_auth_login_required
 @handle_rpc_exceptions
-def patch(
-    *, table_oid: int, metadata_dict: SettableTableMetaData, database_id: int, **kwargs
-) -> TableMetaData:
+def set_(
+    *, table_oid: int, metadata: TableMetaDataBlob, database_id: int, **kwargs
+) -> TableMetaDataRecord:
```

Comment:
```suggestion
) -> None:
```

Reply:
Fixed in 3396114bd98f0898b93943ea5c8c75b3fbc6ee67. Thanks!